polling/src/main/java/io/atleon/polling/reactive/PollingHandlerImp.java

```diff
@@ -0,0 +1,71 @@
+package io.atleon.polling.reactive;
+
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+import reactor.core.publisher.Sinks;
+import reactor.core.scheduler.Scheduler;
+import reactor.core.scheduler.Schedulers;
+
+import java.time.Duration;
+import java.util.Collection;
+import java.util.concurrent.ThreadFactory;
+import java.util.concurrent.atomic.AtomicLong;
+
+public class PollingHandlerImp<K extends PollingRecord<V>, V> implements PollingHandler<K, V> {
```

Comment:
Following my previous naming recommendations, I think this guy is the actual `Poller`, as it contains the EventLoop and is the one doing the polling